package/MDAnalysis/coordinates/chain.py

```diff
@@ -80,68 +166,69 @@ class ChainReader(base.ProtoReader):
     """
     format = 'CHAIN'
 
-    def __init__(self, filenames, **kwargs):
-        """Set up the chain reader.
+    def __init__(self, filenames, skip=1, dt=None, continuous=False, **kwargs):
+        """Set up the chain reader. This can also be used similar to `gmx trjcat` to
```

Comment:
I would put the sentence "This can also be used similar to `gmx trjcat` to..." on a separate line as it is not part of the single-line "subject", especially as not everyone might be familiar with `gmx`.